### PR TITLE
Json format output

### PR DESCRIPTION
**Pull request by dpwspoon**
starting with just counters, queues, and routes

### CODE REVIEW

**Comment by dpwspoon** [comment]
*Sample output*  

-t queues | jq -s .
```json
[
  {
    "kafka-sse.streams": 0
  },
  {
    "kafka-sse.throttle": 0
  },
  {
    "http2.streams": 0
  },
  {
    "http2.throttle": 0
  }
]
```

-t counters | jq -s .
```json
[
  {
    "nukleus": "sse",
    "name": "routes",
    "value": 1
  },
  {
    "nukleus": "sse",
    "name": "acquires",
    "value": 0
  },
  {
    "nukleus": "sse",
    "name": "releases",
    "value": 0
  }
]
```

-t routes | jq -s .
```
[
  {
    "$nukleus": "sse",
    "$id": 0,
    "role": "SERVER",
    "source": "http2",
    "sourceRef": 2,
    "target": "kafka-sse",
    "targetRef": -9223372036854776000,
    "authorization": 0
  },
  {
    "$nukleus": "kafka-sse",
    "$id": 0,
    "role": "PROXY",
    "source": "sse",
    "sourceRef": -9223372036854776000,
    "target": "kafka",
    "targetRef": 1,
    "authorization": 0
  },
  {
    "$nukleus": "kafka-sse",
    "$id": 1,
    "role": "PROXY",
    "source": "sse",
    "sourceRef": -9223372036854776000,
    "target": "kafka",
    "targetRef": 1,
    "authorization": 0
  }
]
```

File: pom.xml [hunk 1]
```diff
@@ -70,6 +71,12 @@
       <artifactId>nukleus</artifactId>
       <version>${nukleus.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.reaktivity</groupId>
+      <artifactId>nukleus-tls.spec</artifactId>
+      <version>${nukleus.tls.spec.version}</version>
+      <scope>provided</scope>
+    </dependency>
```
Comment (jfallows) [inline pom.xml]: Please move this up with the other `.spec` dependencies.

File: src/main/java/org/reaktivity/command/log/internal/LoggableRoutes.java [hunk 2]
```diff
@@ -106,14 +107,45 @@ private int logRoutes(
...
+                    extension = String.format(
+                    "{" +
+                    "\"store\":\"%s\"," +
+                    "\"hostname\":\"%s\"," +
+                    "\"applicationProtocol\":\"%s\"" +
+                    "}",
+                    hostname,
+                    applicationProtocol,
+                    store);
+                }
```
Comment (jfallows) [inline src/main/java/org/reaktivity/command/log/internal/LoggableRoutes.java]: Hoist this to another method returning `extension`, where we can later add support for other nuklei.

**Comment by dpwspoon** [comment]
Feedback applied and changed queue depth to following format

```
[
  {
    "nukleus": "kafka-sse",
    "type": "streams",
    "depth": 0
  },
  {
    "nukleus": "kafka-sse",
    "type": "throttle",
    "depth": 0
  }
]
```

**Pull request merged**